source/renderer/app/components/widgets/WalletTestEnvironmentLabel.js

```diff
@@ -3,24 +3,40 @@ import { defineMessages, intlShape } from 'react-intl';
 import styles from './WalletTestEnvironmentLabel.scss';
 
 const messages = defineMessages({
-  testnetLabel: {
+  staging: {
+    id: 'test.environment.stagingLabel',
+    defaultMessage: '!!!Staging vx',
+    description: 'Label for staging network with version.'
+  },
+  testnet: {
     id: 'test.environment.testnetLabel',
     defaultMessage: '!!!Testnet vx',
     description: 'Label for testnet with version.'
   },
+  development: {
+    id: 'test.environment.developmentLabel',
+    defaultMessage: '!!!Development vx',
+    description: 'Label for development with version.'
+  }
 });
 
-export default class WalletTestEnvironmentLabel extends Component {
+type Props = {
+  network: string,
```

Comment:
@daniloprates let's rather use more strict type definitions here.
Let's declare the `networkType` under type definitions `/source/renderer/app/types/networkTypes.js`
```
export type networkType = 'mainnet' | 'staging' | 'testnet' | 'development';
```
...and then import that type as a definition for the `network` Prop.